### ISSUE TITLE
Refreshing microbadge after a docker push

### ISSUE THREAD

**Issue by lfoppiano** [microscaling/microbadger#11]
Dear all, 
    I have a question, I’ve pushed a new image on the docker hub, version 0.4.1 on https://hub.docker.com/r/lfoppiano/grobid/tags/ but I cannot find it in the relative page of microbadger: https://microbadger.com/images/lfoppiano/grobid. 

According to docker hub the webshook was triggered correctly: 

ac54bb7342e74b9abcc598c996b9e37c  
success
10/03/16 15:26

How long does it take to refresh the microbadger image? 

Thanks
Luca

**Comment by rossf7** [comment]
Hi Luca,
I'm sorry you've hit this problem but thanks for raising it. It looks like a timing problem at our end.

When you call the webhook we put a message on a queue but it's usually picked up immediately. When we refresh the metadata we check the last updated date returned by the DockerHub API. This is so we only call the Registry API and refresh our data if the image has actually changed.

From our logs we checked your image at 15:26 but the last updated date hadn't changed. I just called the webhook again and this time it had changed and we updated your metadata.

We'll do some more investigation and get this timing issue fixed.

Thanks

Ross
